wetterdienst/dwd/forecasts/metadata/parameter.py

```diff
@@ -0,0 +1,127 @@
+from enum import Enum
+
+
+class DWDFcstParameter(Enum):
+    # https://opendata.dwd.de/weather/lib/MetElementDefinition.xml
```

Comment:
Nice!

Reply:
Thanks! This is only a first approach, but it may have happened that I named some parameters different then the ones found in the observations definition.